database/seeders/DatabaseSeeder.php

```diff
@@ -14,5 +14,9 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         // \App\Models\User::factory(10)->create();
+        $this->call([
+            CategorySeeder::class,
+
```

Comment:
As you create the units of measurement table, please create a seeder for it as well and add the seeder here.